lib/modules/platform/gerrit/types.ts

```diff
@@ -34,7 +34,7 @@ export type GerritReviewersType = 'REVIEWER' | 'CC' | 'REMOVED';
 
 export interface GerritChange {
   branch: string;
-  hashtags?: string[];
+  hashtags?: string[]; // for backwards compatibility
```

Comment:
```suggestion
  /**
   * for backwards compatibility
   */
  hashtags?: string[];
```

Use tsdoc comment style